cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -386,13 +384,15 @@ impl ASTNode<Option<cst::Ident>> {
             Some(cst::Ident::Action) => Some(ast::Var::Action),
             Some(cst::Ident::Resource) => Some(ast::Var::Resource),
             Some(ident) => {
-                errs.push(err::ParseError::ToAST(format!(
-                    "expected an identifier, got {ident}"
+                errs.push(err::ParseError::InvalidIdentifier(format!(
+                    "{ident}; expected an identifier"
                 )));
                 None
             }
             None => {
-                errs.push(err::ParseError::ToAST("expected an identifier".to_string()));
+                errs.push(err::ParseError::InvalidIdentifier(
+                    "expected an identifier".to_string(),
```

Comment:
the argument here should be the actual text that failed to parse as an identifier. Or is this another empty-node error and shouldn't be `InvalidIdentifier` at all?